### DISCUSSION TITLE
A few questions for model running

### DISCUSSION THREAD

**Discussion by dinghao1995** [E3SM-Project/E3SM#5603] [E3SM model help]
As a new user of the MPAS-O model from Asia, I have a few questions for you developers: 1. Currently, I use the data provided by MPAS official website to run MPAS-O model, such as EC60to30 or RRS30to10(https://mpas-dev.github.io/), but I don't know how these data are produced, so I would like to ask what is the source of these data? How do you make these data? 2. I also run MPAS-A model, which can be combined with WPS(pre-processing program of WRF model) and driven by FNL data; So what I want to know is, can MPAS-O model also be driven by global reanalysis data similar to FNL like MPAS-A model? I mean the "REAL" simulation. If so, how to achieve it? 3. If the previous problem cannot be realized, how to prepare the data source? And how do I modify the *init file (namelist and streams) to make the driver fields for MPAS-O models?

**Comment by jonbob** [comment]
@dinghao1995 -- unfortunately, E3SM and its components are still in mostly a development mode and there is no one tasked with supporting external users at this point. So as much as we would like to entrain new users, I don't think anyone on our team has much bandwidth for helping at this level.

**Comment by rljacob** [comment]
Moved from Issue to Discussion.
